cognite/powerops/custom_modules/power_model_v1/data_models/containers/BidDocument.container.yaml

```diff
@@ -55,9 +55,15 @@ properties:
     autoIncrement: false
     name: workflowExecutionId
     description: The workflowExecutionId associated with the Bid calculation workflow.
-indexes:
-  deliveryDate:
-    properties:
-      - deliveryDate
-    indexType: btree
-    cursorable: false
+  bidConfiguration:
+    type:
+      type: direct
+    nullable: true
+    autoIncrement: false
+    name: bidConfiguration
```

Comment:
Do we still have priceArea because this container is used across BidDocumentDayAhead (does not have priceArea, because it gets that from the bidConfiguration) and BidDocumentAFRR (or what it's called - probably has priceArea, but I didn't bother to check)? It doesn't really seam meaningful to me to have these two in the same container. The (not very strong?) reason for having these implement the same interface (BidDocument) is standardisation - not that we may be interested in listing both kinds of BidDocuments together (at least I don't see any case where this is relevant/meaningful).

Reply:
I'd say both types of bid documents contain all the same fields, but the only difference is what type a few of the direct relations/multi_edge_connections point to hence why I think it's fair to have it all in one container.

Day Ahead has relations to `bidConfiguration`, `total`, and `partials`. While current AFFR (which probably will need some changes when we actually start on AFFR) has relations to `priceArea` (would think this maybe gets replaced with a similar "configuration" object like we have for Day Ahead) and `bids` (BidRow which to me is "the same" as the relation to BidMatrix for DayAhead)